Adding a fire station

alexblumenfeld opened the pull request.

gallettilance reviewed (changes requested):
Great work! Please make the number of trucks in the station variable (and add a default value to the parameter so that you don't need to change the way the function is called). Also please squash your commits.

alexblumenfeld commented:
I just added a parameter for the number of trucks to firestation.py and squashed my earlier commits. Let me know if there's anything else I should include!

gallettilance commented on 01-python/safety/firestation.py:
```diff
@@ -1,3 +1,14 @@
+    print("|  | []  [] |  |" + "  |________________|" * trucks)
+    print("|  |        |  |" + "    []         []   " * trucks)
+    print("----------------" + "--------------------" * trucks)
+    return
```
Nit: please add a newline here.

alexblumenfeld replied:
Has now been added -- thanks for pointing that out.

gallettilance reviewed (changes requested):
Great work! Almost there
